internal/pkg/crawl/capture.go

```diff
@@ -150,6 +150,18 @@ func (c *Crawl) captureAsset(item *frontier.Item, cookies []*http.Cookie) error
 		c.Frontier.Seencheck.Seen(hash, item.Type)
 	}
 
+	if c.UseHQ {
+		found, err := c.hqSeencheck(item.URL)
+		if err != nil {
+			// potentially allow failure here, since this is only seencheck and HQ could have high load or another issue
```

Comment:
Yes, I think we should just crawl if HQ fails to answer us. Better to be duplicating than loosing stuff.